base/Dockerfile

```diff
@@ -32,7 +34,7 @@ ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
 
 # Install StackStorm, but without UI
 RUN if [ "${ST2_VERSION#*dev}" != "${ST2_VERSION}" ]; then \
-    ST2_REPO=unstable; \
+    ST2_REPO=staging-unstable; \
```

Comment:
I don't know the difference between unstable and staging-unstable. Is switching to staging-unstable still "valid" a year later?

Reply:
We'll need `unstable`.
See more info behind packaging repositories: https://github.com/StackStorm/discussions/wiki/Useful-info-for-Release-Managers#packagecloud-repositories